bluepyopt/ephys/templates/acc/decor_acc_template.jinja2

```diff
@@ -10,7 +10,7 @@
     {%- endif %}
     {%- else %}
     {%- for param in params %}
-    (default ({{ param.name }} {{ param.value }}))
+    (default ({{ param.name }} {{ param.value }} (scalar 1.0)))
```

Comment:
I won't check all of those in detail, but this is correct for a fixed value.
If the tests are green, I'd assume all are correct.

Reply:
The tests are green

Reply:
I'd recommend to additionally check the major examples' notebooks qualitatively by inspection (not everything is run as part of the test suite).